docs: Updated the documentation to use new lexicon - Fixes #28

CynthiaPeter opened the pull request:
<!--
Please make sure you've read and understood our contributing guidelines in CONTRIBUTING.md

If this is a bug fix, make sure your description includes "fixes #xxxx", or
"closes #xxxx"

Please provide the following information:
-->

**- What I did**
This PR fixes issues with #28 

**- How I did it**
I updated the terms with new lexicons.
@platform to atPlatform.
@sign to atSign
secondary server to atServer
root server to atDirectory

**- How to verify it**

**- Description for the changelog**
<!--
Write a short (one line) summary that describes the changes in this
pull request for inclusion in the changelog:
-->
Updated the documentation to use the new lexicon.

CynthiaPeter committed:
docs: Updated the documentation to use new lexicon #28

cpswan commented on specification/at_protocol_specification.md:
````diff
@@ -694,81 +694,81 @@ data:
 
 
-```error:AT0007-Secondary Server not found.```
+```error:AT0007-atServer not found.```
````
The code hasn't been updated yet, so it still says `Secondary server not found`

cpswan replied:
I've raised https://github.com/atsign-foundation/at_tools/issues/275 to get this changed, but that might not happen straight away.

cpswan replied:
The docs here need to reflect the actual error message until it gets fixed in the code.

cpswan commented on specification/at_protocol_specification.md:
```diff
@@ -1,11 +1,11 @@
    <strong>Subject</strong>
    </td>
-   <td>@protocol specification
+   <td>atProtocol specification
```
Please bump version number and date.

CynthiaPeter replied:
I updated this to Feb, 6, 2023

cpswan commented on specification/at_protocol_specification.md:
````diff
@@ -694,81 +694,81 @@ data:
+If the other atServer on which the lookup needs to be performed is down then the atServer should return the following error and keep the connection alive.
 
-```error:AT0007-Secondary Server not found.```
+```error:AT0007-atServer not found.```
````
Same issue as noted above.

CynthiaPeter replied:
Fixed this too

cpswan replied:
Likewise here.

cpswan reviewed (changes requested):
Thanks @CynthiaPeter mostly LGTM. Just a few tweaks before this can be merged.

cpswan commented:
@CynthiaPeter when do you expect to be able to make the changes I requested a couple of weeks ago?

CynthiaPeter commented:
Oh, I missed this.
I’ll get to it now.

Apologies.

On Mon, 6 Feb 2023 at 17:19, Chris Swan ***@***.***> wrote:

> @CynthiaPeter <https://github.com/CynthiaPeter> when do you expect to be
> able to make the changes I requested a couple of weeks ago?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/atsign-foundation/at_protocol/pull/34#issuecomment-1419353591>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIAG7VFXAI4QJ6NPGNNUFNDWWEQB5ANCNFSM6AAAAAAUFKFS6I>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
*Cynthia Peter,*
*Twitter* : ***@***.****

*LinkedIn: Cynthia Peter <https://www.linkedin.com/in/cynthiapeter/>*

CynthiaPeter committed:
docs: Made changes to the reviewed document

xavierchanth commented:
@cpswan please merge and close #28 if you are happy with the changes

CynthiaPeter committed:
docs: Fixed the highlighted issues

We'll implement changes for lines 762 and 734 when the actual codes have been updated.

If this is something I can update, let me know how I can help.

cpswan commented:
Thanks @CynthiaPeter :)

CynthiaPeter commented:
Thank you, @cpswan.

I'm sorry for missing the feedback and dragging the PR for weeks.
